### PR TITLE
Patch links — miscellanous (1)

### PR DESCRIPTION
**Pull request by alesanter**

### CODE REVIEW

File: symplyphysics/laws/chemistry/cross_section_of_interaction_in_model_of_quasirigid_spheres.py [hunk 1]
```diff
@@ -16,6 +16,10 @@
 ## g - the cross-sectional area of the interaction of particles,
 ## r - the distance of the greatest convergence of two colliding particles.
 
+# Links: Wikipedia, second formula <https://en.wikipedia.org/wiki/Cross_section_(physics)#Collision_among_gas_particles>
+
+# FIXME: shouldn't it be `(2*r)^2` as per the formula in the link?
```
Comment (blackyblack) [inline symplyphysics/laws/chemistry/cross_section_of_interaction_in_model_of_quasirigid_spheres.py:21]: It is probably due to distance between colliding particles = 2 * radius of a particle.

File: symplyphysics/laws/chemistry/free_path_of_atomic_particles_in_gaseous_medium.py [hunk 2]
```diff
@@ -13,6 +13,12 @@
 ## P - pressure,
 ## g - the cross section of the interaction of a traveling atom and a gas atom.
 
+# Links: Wikipedia, the fourth formula <https://en.wikipedia.org/wiki/Mean_free_path#Kinetic_theory_of_gases>
+
+# FIXME: hasn't an additional `sqrt(2)` factor been left out in the denominator?
```
Comment (blackyblack) [inline symplyphysics/laws/chemistry/free_path_of_atomic_particles_in_gaseous_medium.py:18]: Looks like a bug. If possible, find some real numbers to write proper test.

File: symplyphysics/laws/chemistry/velocity_of_directional_motion_of_charged_particles_in_gas.py [hunk 3]
```diff
@@ -12,6 +12,8 @@
 ## E - electric intensity,
 ## p - pressure.
 
+# Links: derivable from this assuming `mu = b0 / p` <https://en.wikipedia.org/wiki/Electron_mobility#>
```
Comment (blackyblack) [inline symplyphysics/laws/chemistry/velocity_of_directional_motion_of_charged_particles_in_gas.py:15]: Just give a link, without conditions.

File: symplyphysics/laws/condensed_matter/concentration_of_intrinsic_charge_carriers.py [hunk 4]
```diff
@@ -9,13 +9,13 @@
...
-
-#. `Intrinsic carrier concentration <https://www.universitywafer.com/intrinsic-carrier-concentration.html>`_.
-
 **Conditions:**
 
 #. There are no external influences, such as lighting, electric field, etc.
+
+**Links:**
+
+#. `Intrinsic carrier concentration <https://www.universitywafer.com/intrinsic-carrier-concentration.html>`_.
```
Comment (blackyblack) [inline symplyphysics/laws/condensed_matter/concentration_of_intrinsic_charge_carriers.py:18]: Add source name.

File: symplyphysics/laws/condensed_matter/current_density_from_mobility.py [hunk 5]
```diff
@@ -24,6 +24,8 @@
 ## u2 - mobility of holes,
 ## E - electric intensity (physical field that surrounds electrically charged particles).
 
+# Links: last formula <https://en.wikipedia.org/wiki/Electron_mobility#Relation_to_current_density>
```
Comment (blackyblack) [inline symplyphysics/laws/condensed_matter/current_density_from_mobility.py:27]:
```suggestion
# Links: Wikipedia, last formula <https://en.wikipedia.org/wiki/Electron_mobility#Relation_to_current_density>
```

File: symplyphysics/laws/condensed_matter/resistance_from_temperature.py [hunk 6]
```diff
@@ -21,6 +20,8 @@
 ## T - temperature,
 ## T0 - 273.15 kelvin degrees.
 
+# Links: similar formula for resistivity <https://byjus.com/physics/resistivity-temperature-dependence/>
```
Comment (blackyblack) [inline symplyphysics/laws/condensed_matter/resistance_from_temperature.py:23]:
```suggestion
# Links: BYJU's, similar formula for resistivity <https://byjus.com/physics/resistivity-temperature-dependence/>
```

File: symplyphysics/laws/chemistry/cross_section_of_interaction_in_model_of_quasirigid_spheres.py (hunk 1 above)
Comment (alesanter) [reply symplyphysics/laws/chemistry/cross_section_of_interaction_in_model_of_quasirigid_spheres.py:21]: You're right, I was confused by the description

Comment (alesanter) [reply symplyphysics/laws/chemistry/cross_section_of_interaction_in_model_of_quasirigid_spheres.py:21]: I'll add a note

File: symplyphysics/laws/chemistry/free_path_of_atomic_particles_in_gaseous_medium.py (hunk 2 above)
Comment (alesanter) [reply symplyphysics/laws/chemistry/free_path_of_atomic_particles_in_gaseous_medium.py:18]: Technically this is a "naive" formula that appears in the derivation of the correct formula, so I get where this comes from. Although we should probably only include the correct one.

Comment (alesanter) [reply symplyphysics/laws/chemistry/free_path_of_atomic_particles_in_gaseous_medium.py:18]: I'll fix this law and test tho.

**Pull request merged**